pkg/asset/manifests/operators.go

```diff
@@ -62,6 +62,7 @@ func (m *Manifests) Dependencies() []asset.Asset {
 		&Proxy{},
 		&Scheduler{},
 		&ImageContentSourcePolicy{},
+		&LoadBalancerUID{},
```

Comment:
should this asset be created in infrastructure.go rather than operators.go?

Reply:
I'll move it. I debated whether to put it there but reasoned that infrastructure.go's main purpose was to create the infrastructure config. However, thinking more about it, it makes sense.